link/src/test/java/com/stripe/android/link/utils/FakeIntentConfirmationInterceptor.kt

```diff
@@ -0,0 +1,50 @@
+package com.stripe.android.link.utils
+
+import com.stripe.android.IntentConfirmationInterceptor
+import com.stripe.android.model.ConfirmPaymentIntentParams
+import com.stripe.android.model.ConfirmStripeIntentParams
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.PaymentMethodCreateParams
+import com.stripe.android.model.StripeIntent
+import kotlinx.coroutines.channels.Channel
+
+internal class FakeIntentConfirmationInterceptor : IntentConfirmationInterceptor {
```

Comment:
Is there no way to share the FakeIntentConfirmationInterceptor?

Reply:
Possible now after I merged https://github.com/stripe/stripe-android/pull/6374.